lib/manageiq/style/cli.rb

```diff
@@ -7,8 +7,16 @@ def self.run
         new.run
       end
 
+      OPTION_DEFAULTS = {
+        :install     => false,
+        :rubocop     => true,
+        :codeclimate => true,
+        :yamllint    => true,
+        :hamllint    => true
```

Comment:
Why default these to `true` here, but `false` [here](https://github.com/ManageIQ/manageiq-release/pull/301/files#diff-859bf08b7807002b74ce752e64deea785ef11b1d73fccd250fa5a53df4a4edf2R16)